README.md

```diff
@@ -5,18 +5,79 @@ This repository is part of the Simularium project ([simularium.allencell.org](ht
 - [simularium-viewer](https://github.com/allen-cell-animated/simularium-viewer) - NPM package to view Simularium trajectories in 3D
 - [simularium-website](https://github.com/allen-cell-animated/simularium-website) - Front end website for the Simularium project, includes the Simularium viewer
 
+<br/>
+
+---
 ---
+<br/>
 
 # SimulariumIO
-
 [![Build Status](https://github.com/allen-cell-animated/simulariumio/workflows/Build%20Master/badge.svg)](https://github.com/allen-cell-animated/simulariumio/actions)
 [![Documentation](https://github.com/allen-cell-animated/simulariumio/workflows/Documentation/badge.svg)](https://allen-cell-animated.github.io/simulariumio)
 [![Code Coverage](https://codecov.io/gh/allen-cell-animated/simulariumio/branch/master/graph/badge.svg)](https://codecov.io/gh/allen-cell-animated/simulariumio)
 
-Simulariumio converts simulation outputs to the format consumed by the [Simularium viewer](https://simularium.allencell.org/).
+---
+---
+
+<br/>
+
+
+## Convert simulation outputs so they can be visualized in the [Simularium Viewer](https://simularium.allencell.org/).
+
+<br/>
+
+# 1. Convert your data
+Save your data as a .simularium file (see Jupyter notebook [examples](examples/)):
+
+```python
+import numpy as np
+from simulariumio import TrajectoryConverter, TrajectoryData, MetaData, AgentData, ScatterPlotData
+
+converter = TrajectoryConverter(
+    TrajectoryData(
+        meta_data=MetaData(
+            box_size=BOX_SIZE_NUMPY_ARRAY,
+            trajectory_title="Model A with parameter set 1",
+        ),
+        agent_data=AgentData(
+            DATA_FOR_OBJECTS_MOVING_OVER_TIME
+        ),
+    )
+)
+converter.add_plot(
+    ScatterPlotData(
+        title="Something measured",
+        xaxis_title="Time (s) or something else",
+        yaxis_title="Some measurement (units)",
+        xtrace=X_VALUES_NUMPY_ARRAY,
+        ytraces={
+            "Agent 1" : Y_VALUES_NUMPY_ARRAY,
+        },
+    )
+)
+converter.write_JSON("output_file_name")
+```
+
+<br/>
+
+# 2. Load in the [Simularium Viewer](https://simularium.allencell.org/viewer)
+Load your file in the Simularium Viewer at https://simularium.allencell.org/viewer to interactively rotate and play back your data:
+
+![Loading a file in the Simularium Viewer](/examples/img/drag_drop.gif)
+
+<br/>
+
+# 3. Share by URL 
+Upload your file to a public Dropbox or Google Drive folder or an AWS S3 bucket, and use the URL https://simularium.allencell.org/viewer?trajUrl= [link to your file]
```

Comment:
for clarity, 
```suggestion
Upload your file to a public Dropbox or Google Drive folder or an AWS S3 bucket, and use the URL https://simularium.allencell.org/viewer?trajUrl=[LINK_TO_YOUR_FILE]
```
And maybe have an example that works (a link to one of our trajectories)

Reply:
when I don't leave a space between `=[` the opening bracket gets highlighted as part of the URL, do you know how to escape it? I tried prepending `\` but that didn't work. maybe it doesn't matter

Reply:
I think that's fine, I would probably even remove the brackets so people don't think you need them. I know that's common in code, bu in this case it's just a string